.github/workflows/update-test-times.yml

```
@@ -0,0 +1,33 @@
+name: Update test times
+
+on:
+  schedule:
+    # Every day at 3:05am EDT
```

```suggestion
    # Every day at 3:05am UTC
```
nit--this is probably UTC